CyclopsNuclearReactor/uGUI_Patches.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace CyclopsNuclearReactor
+{
+    using Harmony;
+    using System.Collections.Generic;
+
+    [HarmonyPatch(typeof(uGUI_InventoryTab))]
+    [HarmonyPatch("OnOpenPDA")]
+    internal class UGUI_InventoryTab_OnOpenPDA_Patcher
+    {
+        [HarmonyPostfix]
+        internal static void Postfix(uGUI_InventoryTab __instance)
+        {
+            // This event happens whenever the player opens their PDA.
+            // We will make a series of checks to see if what they have opened is the Cyclops Bioreactor item container.
+
+            if (__instance == null)
+                return; // Safety check
+
+            if (!Player.main.IsInSub() || !Player.main.currentSub.isCyclops)
+                return; // If not in Cyclops then all is irrelevant
+
+            uGUI_Equipment equipmentUI = __instance.equipment;
+
+            if (equipmentUI == null)
+                return; // Not an equipment container
+
+            var equipment = (Equipment)AccessTools.Field(typeof(Equipment), "equipment").GetValue(equipmentUI);
```

Comment:
This is wrong somehow.  
Double-check all the Reflection calls in this patch method.